precompiles/staking/events.go

```diff
@@ -122,6 +124,46 @@ func (p Precompile) EmitAllowanceChangeEvent(ctx sdk.Context, stateDB vm.StateDB
 	return nil
 }
 
+// EmitCreateValidatorEvent creates a new ctreate valdator event emitted on a CreateValidator transaction.
+func (p Precompile) EmitCreateValidatorEvent(ctx sdk.Context, stateDB vm.StateDB, msg *stakingtypes.MsgCreateValidator, delegatorAddr common.Address) error {
+	// Prepare the event topics
+	event := p.ABI.Events[EventTypeCreateValidator]
+	topics, err := p.createStakingTxTopics(3, event, delegatorAddr, msg.ValidatorAddress)
```

Comment:
this needs to be updated with the new address type for the validator

Reply:
done. 
Please remind your team that other events such as event Delegate also need to be updated. This is the modified event。
<img width="1584" alt="image" src="https://github.com/evmos/evmos/assets/9688029/29480e16-e820-41d0-9ad8-f225ee226b49">

Reply:
done !